.github/workflows/deploy.yaml

```diff
@@ -172,16 +170,11 @@ jobs:
           mv dist/MDAnalysisTests-* testsuite/dist
 
       - name: upload_source_and_wheels
-        uses: pypa/gh-action-pypi-publish@v1.5.0
-        with:
-          user: __token__
-          password: ${{ secrets.PYPI_API_TOKEN_SRC }}
+        uses: pypa/gh-action-pypi-publish@v1.18.10
```

Comment:
Is the MDA deployment so complicated that we can't use our action https://github.com/MDAnalysis/pypi-deployment ?

Reply:
cibuildwheels deployments don't work via pypi-deployment right now - that's a bit more complicated so I've not added that branch yet (it's a bit more messy because you usually want to parallelise at the level of runners for each flavour you build wheels for - you don't fully have to though, you can just trust the docker container to get it right).

One day I'll get that done, it's just not been a burning priority (although the amount of cython packages we're maintinaing is starting to grow).

What I can't stuff in a reusable action though is the arm builds (using Cirrus), those will need to exist everywhere we need those builds (currently just here).

Reply:
Thanks for the explanations!

Reply:
Extra info for anyone reading this and going "but why cibuildwheels here and not everywhere else" (it's a recurring question we should document) - we use cibuildwheels to build anything that can't do a pure Python wheel (i.e. uses Cython).

Reply:
Do we have a "maintainer FAQ" somewhere — wiki or NF drive?

Reply:
Unfortunately not, we have barbones deployment docs on the userguide but yeah there's just not been enough of my time to add docs for everything - that's what we applied for that SDG for though so fingers crossed!